Renderdoc fails to extract and run linux

Running `runClientRenderDoc` on linux fails with this error
Can be replicated using the current version - ([ae0ae72](https://github.com/FabricMC/fabric-example-mod/tree/ae0ae725cb4ec0c37644a6aec1fe3c122b9ad92d)) - of fabric-example-mod
```java
Execution failed for task ':extractRenderDoc'.
> Cannot convert the provided notation to a File: task ':downloadRenderDoc' property 'output'.
  The following types/formats are supported:
    - A String or CharSequence path, for example 'src/main/java' or '/usr/include'.
    - A String or CharSequence URI, for example 'file:/usr/include'.
    - A File instance.
    - A Path instance.
    - A Directory instance.
    - A RegularFile instance.
    - A URI or URL instance of file.
    - A TextResource instance.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
For more on this, please refer to https://docs.gradle.org/9.2.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
BUILD FAILED in 29s
10 actionable tasks: 6 executed, 4 up-to-date
```

--- a/src/main/java/net/fabricmc/loom/task/LoomTasks.java
+++ b/src/main/java/net/fabricmc/loom/task/LoomTasks.java
@@ -237,9 +237,9 @@ public abstract class LoomTasks implements Runnable {
 			task.setGroup(Constants.TaskGroup.FABRIC);
 
 			if (operatingSystem.isWindows()) {
-				task.from(getProject().zipTree(downloadRenderDoc.map(DownloadTask::getOutput)));
+				task.from(getProject().zipTree(downloadRenderDoc.flatMap(DownloadTask::getOutput)));
 			} else {
-				task.from(getProject().tarTree(downloadRenderDoc.map(DownloadTask::getOutput)));
+				task.from(getProject().tarTree(downloadRenderDoc.flatMap(DownloadTask::getOutput)));
 			}
 
 			task.into(getProject().getLayout().getBuildDirectory().dir("renderdoc"));

